security/oidc-client-mutual-tls/src/test/java/io/quarkus/ts/security/oidcclient/mtls/Pkcs12OidcMtlsIT.java

```diff
@@ -15,14 +15,12 @@
  * Keystore file type is automatically detected in following tests by its extension in quarkus-oidc.
  * Extension declared here is used by tests only.
  */
-@Tag("fips-incompatible")
+@Tag("fips-incompatible") // OpenJDK 11
 @QuarkusScenario
 public class Pkcs12OidcMtlsIT extends KeycloakMtlsAuthN {
 
-    //TODO Remove workaround after Keycloak is fixed https://github.com/keycloak/keycloak/issues/9916
-    @KeycloakContainer(command = KC_DEV_MODE_P12_CMD, port = KEYCLOAK_PORT)
+    @KeycloakContainer(image = "quay.io/keycloak/keycloak:22.0", command = KC_DEV_MODE_P12_CMD, port = KEYCLOAK_PORT)
```

Comment:
There is no good reason to use Keycloak 19 as main is on 22, Ghost is on 21 (subject to change) and the last productized version is 18. Please change default to 22 and release framework. If we hardcode images like that, we need to remember changing it on several places, therefore we should have a good reason to set one specific version to one test. Changing default has advantage of using one image everywhere, therefore less images going to be pulled (less docker memory, less execution time).